setup.py

```diff
@@ -1,13 +1,85 @@
+import os
+import platform
+import sys
 from pathlib import Path
 
+from pkg_resources import parse_version
 from setuptools import find_packages, setup
 
 
+def get_decoder_pkg_url(version):
+    is_arm = 'arm' in platform.machine()
+    is_mac = 'darwin' in sys.platform
+    is_64bit = sys.maxsize > (2**31 - 1)
+
+    if is_arm:
+        tc_arch = 'arm64-ctc' if is_64bit else 'arm-ctc'
+    elif is_mac:
+        tc_arch = 'osx-ctc'
+    else:
+        tc_arch = 'cpu-ctc'
+
+    ds_version = parse_version(version)
+    branch = "v{}".format(version)
+
+    plat = platform.system().lower()
+    arch = platform.machine()
+
+    if plat == 'linux' and arch == 'x86_64':
+        plat = 'manylinux1'
+
+    if plat == 'darwin':
+        plat = 'macosx_10_10'
+
+    is_ucs2 = sys.maxunicode < 0x10ffff
+    m_or_mu = 'mu' if is_ucs2 else 'm'
+
+    pyver = ''.join(str(i) for i in sys.version_info[0:2])
+
+    return 'ds_ctcdecoder @ https://community-tc.services.mozilla.com/api/index/v1/task/project.deepspeech.deepspeech.native_client.{branch_name}.{tc_arch_string}/artifacts/public/ds_ctcdecoder-{ds_version}-cp{pyver}-cp{pyver}{m_or_mu}-{platform}_{arch}.whl'.format(
```

Comment:
We could likely get the root URL from `DECODER_ARTIFACTS_ROOT` in CI and this way it should work for all cases, even bumping version.